pkg/model/application.go

```diff
@@ -81,3 +81,11 @@ func (a *Application) IsOutOfSync() bool {
 func IsApplicationConfigFile(filename string) bool {
 	return filename == DefaultApplicationConfigFilename || strings.HasSuffix(filename, applicationConfigFileExtention)
 }
+
+func ToApplicationKind(k string) (ApplicationKind, bool) {
+	appKind, ok := ApplicationKind_value[strings.ToUpper(string(k))]
+	if !ok {
+		return -1, false
+	}
+	return ApplicationKind(appKind), true
+}
```

Comment:
IMO, I don't think this kind of logic needs to be abstracted because we use the map generated by proto to get the numeric value of enum everywhere.

Reply:
Enough to get using `model.ApplicationKind_value` in the side of each caller.

Reply:
Yes. Me too. That is what I wanted to say
https://github.com/pipe-cd/pipe/pull/2934#discussion_r767499213